server/src/main/java/io/crate/execution/ddl/tables/TableCreator.java

```diff
@@ -70,20 +67,6 @@ public CompletableFuture<Long> create(BoundCreateTable createTable, Version minN
         var relationName = createTable.tableName();
         CreateTableRequest createTableRequest;
 
-        boolean tableExists = schemas.tableExists(relationName);
-        boolean viewExists = schemas.viewExists(relationName);
-        boolean noOp = false;
-        if (createTable.ifNotExists() && !viewExists) {
-            noOp = tableExists;
-        } else if (tableExists || viewExists) {
-            throw new RelationAlreadyExists(relationName);
-        } else {
-            noOp = false;
-        }
-        if (noOp) {
-            return CompletableFuture.completedFuture(1L);
-        }
```

Comment:
Removed this because there needs to be a check in the master action anyway, and this also masked an error handling issue (see the change in the CreateTableAsPlan)